src/styles/card.scss

```diff
@@ -0,0 +1,91 @@
+$second-text-color: #616070;
+$button-color: #00acdc;
+
+.card {
+  margin: 0;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__photo {
+    margin: 32px 19px 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+  }
+
+  &__name--position {
+    margin: 0 16px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $second-text-color;
+  }
+
+  &__code--position {
+    margin: 4px 16px 0 ;
```

Comment:
Ss you can see you are repeating margin 16px on x axis in `name `in `code`. You need to set paddings 16px for all items inside `card`